src/Facebook/InstantArticles/Transformer/Rules/GlobalRule.php

```diff
@@ -136,6 +139,21 @@ public function apply($transformer, $instantArticle, $node)
             $header->withTime(Time::create(Time::PUBLISHED)->withDatetime($timePublished));
         }
 
+        // Treats Header Image
+        $articleHeaderImageURL = $this->getProperty(self::PROPERTY_GLOBAL_HEADER_IMAGE, $node);
+        if ($articleHeaderImageURL) {
+            $header->withCover(Image::create()->withURL($articleHeaderImageURL));
+        } else {
```

Comment:
Maybe we don't need to add a warning here as the cover image is optional?

Reply:
Makes sense. I'll remove that block of code